t/discovery/consul2.t

```diff
@@ -145,7 +145,7 @@ location /consul3 {
     "PUT /consul2/deregister/service_a2",
     "PUT /consul3/deregister/service_a1",
     "PUT /consul3/deregister/service_a2",
-    "PUT /consul1/register\n" . "{\"ID\":\"service_a1\",\"Name\":\"service_a\",\"Tags\":[\"primary\",\"v1\"],\"Address\":\"127.0.0.1\",\"Port\":30511,\"Meta\":{\"service_a_version\":\"4.0\"},\"EnableTagOverride\":false,\"Weights\":{\"Passing\":10,\"Warning\":1}}",
+    "PUT /consul1/register\n" . "{\"ID\":\"service_a1\",\"Name\":\"service_a\",\"Tags\":[\"primary\",\"v1\"],\"Address\":\"127.0.0.1\",\"Port\":30511,\"Meta\":{\"service_a_version\":\"4.0\", \"weight\":\"100\"},\"EnableTagOverride\":false,\"Weights\":{\"Passing\":10,\"Warning\":1}}",
```

Comment:
Based on my questions regarding the changes related to `weight`, I think the testing for this part needs to be reconsidered.